input/community/speakers/charles-nurse.md

```diff
@@ -0,0 +1,21 @@
+---
+Title: Charles Nurse
+Pronouns: he/him
+Location: Chilliwack, BC, Canada
+Email: charlesnurse@gmail.com
+Language:
+  - English
+Topics:
+  - .NET
+  - ASP.NET
+  - Big Data
+  - Blazor
+  - Web Development
+Blog: http://www.charlesnurse.com
+Twitter: https://twitter.com/cnurse
+LinkedIn: https://www.linkedin.com/in/charlesnurse/
+Sessionize: https://sessionize.com/charles-nurse
+---
+Charles Nurse is a Pluralsight Author, an 11-time Microsoft MVP (2007-2018) and an ASPInsider (since 2008).   He is a successful .NET Architect and a Graph enthusiast, and he has led development teams on both Open Source projects and large Enterprise projects.
+
+He has spoken at many major conferences including the Software Developers Conference (SDC) (www.sdc.nl),   DevConnections (www.devconnections.com),  DevTeach (www.devteach.com), VSLive (www.vslive.com), Prairie DevCon(www.prairiedevcon.com) and DevReach(www.DevReach.com).    He is a frequent speaker at local User Groups and Code Camps, including .NET BC, VBug, Vancouver TechFest and Microsoft Developer days.
```

Comment:
```suggestion
He has spoken at many major conferences including the Software Developers Conference (SDC) (https://www.sdc.nl),   DevConnections (https://www.devconnections.com),  DevTeach (https://www.devteach.com), VSLive (https://www.vslive.com), Prairie DevCon(https://www.prairiedevcon.com) and DevReach (https://www.DevReach.com).    He is a frequent speaker at local User Groups and Code Camps, including .NET BC, VBug, Vancouver TechFest and Microsoft Developer days.
```

Adding https links as requested. @cnurse, if you accept these suggestions, that should validate this PR I think and allow it to be merged.